sound/hda/intel-dsp-config.c

```diff
@@ -558,8 +558,29 @@ static const struct config_entry *snd_intel_dsp_find_config
 			int i;
 
 			for (i = 0; i < table->codec_hid->num_codecs; i++)
-				if (acpi_dev_present(table->codec_hid->codecs[i], NULL, -1))
-					break;
+				if (acpi_dev_present(table->codec_hid->codecs[i], NULL, -1)) {
```

Comment:
Btw, you can reduce indentation level, by inverting this condition (i.e. replacing 'break' by 'continue' and inverting the conditional in the original code).

Reply:
good, suggestion, done!